alert/app.py

```diff
@@ -1,7 +1,4 @@
 from flask import Flask
-from flask_bootstrap import Bootstrap
```

Comment:
We will include Bootstrap through JavaScript/CSS links in the HTML. I'm not a big fan of `flask_bootstrap` and we don't want the Flask code knowing too much about the HTML.

Reply:
No worries. This makes more sense than `flask_bootstrap`. I'll probably do it like this from now :)